project-base/tests/FrontendApiBundle/Functional/Product/ProductTest.php

```diff
@@ -67,18 +67,33 @@ public function testProductDetailWithAllAttributesByUuid(): void
                     },
                     stockQuantity,
                     categories {
-                      name
+                        name
                     },
                     flags {
-                      name, rgbColor
+                        name, 
+                        rgbColor
                     },
                     price {
-                      priceWithVat,
-                      priceWithoutVat,
-                      vatAmount
+                        priceWithVat,
+                        priceWithoutVat,
+                        vatAmount
                     },
                     brand {
-                      name
+                        name
+                    },
+                    accessories {
+                        name
+                    },
+                    isSellingDenied,
+                    description,
+                    orderingPriority
+                    parameters {
+                        parameter {
+                            name
+                        }
+                        value {
+                            text
+                        }
```

Comment:
Are you sure this structure is the one we want?

What do you think about having it like this:
```
parameters {
    name
    values {
        text
    }
}
```
Whould not it be better especially when we start using it for displaying filter?

Reply:
There can be only one parameter of same type per product, so this makes no sense

Reply:
I agree with @s3tezsky here. I think that the usage with values inside parameter will be a lot better on FE (especially in filter) and the structure won't be as confusing and will follow the "GraphQL way" of relations

Reply:
> There can be only one parameter of same type per product, so I do not see need for that

This is sad, as you may want to have more values to one parameter. I guess we should not tighten us by introducing this structure in FEAPI too.

E.g.: When you have a wifi router you will have parameter called frequency where you can have both of 2,4GHz and 5GHz and so on.

Reply:
Guys, I understand, what you are saying and basically I agree. But I see there possible confusion about returning different type, than actual used in application. 

For example, we do know, that many of e-commerce solutions uses multiple warehouses and wants to return stock per warehouse, but we return only integer of stock quantity, because it is current implementation. 

So I see this as subject of discussion and would like to see at least @grossmannmartin opinion and @sslt opinion would be also appreciated.

Reply:
I have updated structure as requrested